generatedocs/data/params.json

```diff
@@ -1,4 +1,10 @@
 {
+    "destinationspace.destinationspaceid": [
+        "Destination space, specified by ID."
```

Comment:
Maybe change to "ID of the destination space."?

Reply:
Sure, I could do that. I think maybe the current wording more suggests that only one of them should be used, as they specify the same thing in two different ways.

Reply:
Your wording seems to suggest to me that both should be typed. Agree/disagree? It's up to you :)

Reply:
You've got a point. 
Maybe we should go with "Specify destination space by ID." and "Specify destination space by name."?

Reply:
I like that one